GeometryDashAPI/Serialization/TypeDescriptor.cs

```diff
@@ -199,7 +201,7 @@ private static (SetterInfo<T>[], int baseIndex, Dictionary<string, int> mappings
         {
             var keys = new HashSet<string>();
             var maxKeyValue = 0;
-            Dictionary<string, int> mappings = null;
+            Dictionary<string, int> mappings = new();
```

Comment:
I think it should be null for avoid allocation useless dictionary for every type  
But you're right, it's can cause NRE. I've fix it with null check in `Create` method

Reply:
Okay